### DISCUSSION TITLE
Give Admins the ability to remove a flag from a comment

### DISCUSSION THREAD

**Discussion by michael-tharrington** [forem/forem#20489] [Feature Requests]
Admins currently do not have the ability to remove a flag from a comment. This is an important feature to be able to undo mistakes.

**Comment by michael-tharrington** [comment]
I think it would make most sense to be able to go to navigate to a comment’s admin page (e.g. https://dev.to/michaeltharrington/comment/2bpnl/moderate) and be able to take action to remove any flags on the comment there. It would also be handy to have a history of a comment being edited. This way, if someone tried to gaslight somebody by changing what they had written we could review it.

**Comment by Oxana1987** [comment]
Everything is working fine
